src/Core/Style.js

```diff
@@ -0,0 +1,113 @@
+import { FEATURE_TYPES } from 'Core/Feature';
+
+function rgba2rgb(orig) {
+    if (orig && orig.includes('rgba')) {
+        const rgb = orig.replace(/\s/g, '').match(/^rgba?\((\d+),(\d+),(\d+),(\d\.\d)/i);
+        return (rgb && rgb.length === 5) ? { color: `rgb(${rgb[1]}, ${rgb[2]}, ${rgb[3]})`, opacity: rgb[4] } : { style: orig, opacity: 1.0 };
+    } else if (orig && orig.includes('hsla')) {
+        const hsl = orig.replace(/\s/g, '').match(/^hsla?\((\d+),(\d+)%,(\d+)%,(\d\.\d)/i);
+        return (hsl && hsl.length === 5) ? { color: `rgb(${hsl[1]}, ${hsl[2]}, ${hsl[3]})`, opacity: hsl[4] } : { style: orig, opacity: 1.0 };
```

Comment:
You can't say that `h=r` `s=g` and `l=b`, some conversion needs to be done !

Reply:
oopps